docs/content/doc/installation/from-source.en-us.md

```diff
@@ -182,7 +179,7 @@ GOOS=linux GOARCH=arm64 make build
 Cross-build Gitea for Linux ARM64, with recommended build tags:
 
 ```
-CC=aarch64-unknown-linux-gnu-gcc GOOS=linux GOARCH=arm64 TAGS="bindata sqlite sqlite_unlock_notify" make build
+CC=aarch64-unknown-linux-gnu-gcc GOOS=linux GOARCH=arm64 TAGS="bindata" make build
```

Comment:
Also good to remove any reference to CGO? Like providing C profile here.

Reply:
If some users want to use some C modules like `pam`, then CGO is still needed.

Reply:
I was under the impression the pam module didn't use CGO as well.

Reply:
Yes, we use https://github.com/msteinert/pam which is CGO :pensive:

Reply:
> If some users want to use some C modules like `pam`, then CGO is still needed.

But there is a build tag for pam

Reply:
> > If some users want to use some C modules like `pam`, then CGO is still needed.
> 
> But there is a build tag for pam

But this document is showing users how to set the C compiler if the CGO is needed when cross-build